hotsos/core/plugins/storage/ceph.py

```diff
@@ -526,6 +526,10 @@ def bluefs_oversized_metadata_osds(self):
         for device in self.osd_df_tree['nodes']:
             if device['id'] >= 0:
                 meta_kb = device['kb_used_meta']
+                # Util under under 2G are ignored (treated as non-issue).
+                # See #434 for relevant info.
+                if meta_kb < (2 * 1024 * 1024):
```

Comment:
2 G is too small to detect an oversized META usage case, see https://github.com/canonical/hotsos/issues/434#issuecomment-1279652013

Reply:
I think you've mistaken the 2G as the absolute hard limit. We already test whether the meta util is > 5% and warn on that.
Which produces a lot of false positives, necessitating this tweak.

Reply:
Ah, it's just a low limit, that makes sense.